PackageInfo.g

```diff
@@ -70,14 +70,6 @@ ArchiveURL      := Concatenation( ~.SourceRepository.URL,
                                  "/", ~.PackageName, "-", ~.Version ),
 ArchiveFormats := ".tar.gz",
 
-AbstractHTML :=
-  "The <span class=\"pkgname\">kbmag</span> package is a\
-       <span class=\"pkgname\">GAP</span> interface to some `C' programs\
-   for running the Knuth-Bendix completion program on finite semigroup,\
-   monoid or group presentations, and for attempting to compute  automatic\
-   structures of finitely presented groups",
-
-
 PackageDoc := rec(
   BookName  := "kbmag",
   ArchiveURLSubset := ["doc", "htm"],
```

Comment:
This line must be adjusted -- there is no need for a `htm` directory anymore. The `HTMLStart` field also needs to be adjusted. And all files containing `htm` should be inspected. The `doc/make_doc` script can also be removed.